sdk/compute/Azure.ResourceManager.Compute/tests/SessionRecords/DiskAccessOperationsTests/Delete().json

```diff
@@ -362,7 +362,7 @@
       "ResponseHeaders": {
         "Azure-AsyncOperation": "https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/providers/Microsoft.Compute/locations/westus2/DiskOperations/d24f137b-d3e7-48fc-a775-d91b030293af?p=4bf15084-9f45-4a17-82a6-013632a884fb\u0026api-version=2021-08-01",
         "Cache-Control": "no-cache",
-        "Content-Length": "4",
```

Comment:
If this was manually updated isn't this going to fail again if someone re-records?

Reply:
We will be fixing this in the Test Proxy, so if they re-record before it is fixed then yes.

Reply:
I would prefer to wait for that and validate that assumption before merging in.

Reply:
Submitted https://github.com/Azure/azure-sdk-tools/pull/2634